STYLES/styles.css

```diff
@@ -0,0 +1,11 @@
+
+h1 {
+    color: orange;
+    font-family: verdana;
+    font-size: 300%;
+}
+p  {
+    color: green;
+    font-family: courier;
+    font-size: 160%;
+}
```

Comment:
If you're including an ordered list, you may want to do some fun things when styling your list items (`<li>`). Check out this resource for some ideas: [CSS Lists](https://www.w3schools.com/css/css_list.asp)